tests/runner.js

```diff
@@ -1,8 +1,5 @@
 'use strict';
 
-const captureExit = require('capture-exit');
-captureExit.captureExit();
```

Comment:
I hope we don't need this after we abstracted `process` in `MockProcess`(I may be wrong)